docs/detections/about-rules.asciidoc

```diff
@@ -43,6 +43,8 @@ TIP: You can also use value lists as the indicator match index. See <<indicator-
 * <<create-new-terms-rule, *New terms*>>: Generates an alert for each new term detected in source documents within a specified time range. You can also detect a combination of up to three new terms (for example, a `host.ip` and `host.id` that have never been observed together before).
 
 * <<create-esql-rule, *ES|QL*>>: Searches the defined indices and creates an alert when results match an {ref}/esql.html[Elasticsearch Query Language (ES|QL)] query.
++
+NOTE: {esql} features are turned on by default and are controlled by the {kibana-ref}/advanced-options.html#kibana-general-settings[`enableESQL`] advanced setting.
```

Comment:
```suggestion
NOTE: {esql} is enabled by default in {kib}. It can be
disabled using the `enableESQL` setting from the
{kibana-ref}/advanced-options.html[Advanced Settings].
+
This will hide the {esql} user interface from various applications.
However, users will be able to access existing {esql} artifacts like saved searches and visualizations.
```